tests/e2e/sqlalchemy/test_basic.py

```diff
@@ -237,7 +238,11 @@ class SampleObject(base):
         SampleObject.name.in_(["Bim Adewunmi", "Miki Meek"])
     )
 
-    output = [i for i in session.scalars(stmt)]
+    if sqlalchemy_1_3():
```

Comment:
Whoops. This change belongs in a separate commit. Basically: version 1.3 doesn't have the `.scalars` method whereas 1.4 does.